packages/taquito/src/contract/rpc-contract-provider.ts

```diff
@@ -764,6 +765,75 @@ export class RpcContractProvider extends Provider implements ContractProvider, S
 
     return batch;
   }
+
+  /**
+   *
+   * @description Stake founds. Will sign and inject an operation using the current context
```

Comment:
octez-client description `Stake the given amount for the source. The source must be a delegator to be allowed to stake.`